recommender/recommender.py

```diff
@@ -925,11 +926,12 @@ def student_view(self, _context=None):  # pylint: disable=unused-argument
         # load continues to be as-is, pre-generation is not a performance
         # issue. If students make substantially more resources, we may want
         # to paginate, and generate in sets of 5-20 URLs per load.
-        resources = [{'id': r['id'],
+        resources = [{
+                      'id': strip_and_clean_html_elements(r['id']),
                       'title': strip_and_clean_html_elements(r['title']),
-                      "votes": r['upvotes'] - r['downvotes'],
-                      'url': r['url'],
-                      'description': self._get_onetime_url(r['description']),
+                      "votes": strip_and_clean_html_elements(r['upvotes'] - r['downvotes']),
+                      'url': strip_and_clean_url(r['url']),
+                      'description': strip_and_clean_html_elements(self._get_onetime_url(r['description'])),
```

Comment:
What does `_get_onetime_url` mean or do?  Do we have the correct clean method happening here?  Will it work correctly?

Reply:
Looks like `_get_onetime_url` is focusing on `fs` urls:

```    
def _get_onetime_url(self, filename):
        """
        Return one time url for uploaded screenshot

        We benchmarked this as less than 8ms on a sandbox machine.
        """
        if filename.startswith('fs://'):
            return str(self.fs.get_url(filename.replace('fs://', ''), 1000 * 60 * 60 * 10))
        else:
            return filename
```
I don't think that we would want to do `linkify` scrubbing here.

Reply:
@fysheets: Would the following make a little more sense?
```
'description': self._get_onetime_url(strip_and_clean_html_elements(r['description'])),
```

Reply:
Ah, rereading it, yeah it is more clear that way - I'll make that change quickly and then squash.